Vic2ToHoI4/Source/HOI4World/States/HoI4State.cpp

```diff
@@ -99,6 +100,15 @@ void HoI4::State::addNavalBase(int level, int location)
 }
 
 
+void HoI4::State::smashNavalBases()
```

Comment:
speaking of smashing, there aren't any airports or bunkers that need to be deleted, or factories and similar, are there? Just checking.

Reply:
No bunkers anywhere, and airports got removed as a side-effect of removing factories and VPs.